tensorboard/webapp/settings/_views/settings_dialog_component.ng.html

```diff
@@ -55,3 +55,97 @@ <h3>Settings</h3>
     Page size has to be a positive integer.
   </mat-error>
 </div>
+
+<div>
+  <h4>Visualization colors</h4>
+  <div>
+    Customize colors that visualzation such as line chart will use. When there
```

Comment:
typo: visualization, missing `i`  Also should probably be "visualizations"?

FWIW, I think this might read a bit better as something like

> Choose the color palette to use for visualizations (currently only works for the Time Series plugin). When there are more items than the number of colors, it will cycle repeatedly.

In particular - "customize" seems odd since you can't actually specify a custom palette, and specifying "runs" seems a little too specific, because Time Series lets you assign colors to groups that aren't necessarily just 1 run.